config/initializers/wicked_pdf.rb

```diff
@@ -0,0 +1,32 @@
+# WickedPDF Global Configuration
```

Comment:
for PDFs we usually use prawn pdf or react pdf. But i am accepting this. It could be great if you can try out react PDF first.

Reply:
i will look into it thank you for the information